app/src/main/res/values/strings.xml

```diff
@@ -10,196 +10,196 @@
+    <string name="ip_address_val">IP address: %1$s</string>
     <string name="host_val">Host: %1$s</string>
     <string name="protocol_val">Protocol: %1$s</string>
     <string name="whitelist_action">Whitelist…</string>
     <string name="hide">Hide…</string>
     <string name="n_selected">%1$d selected</string>
     <string name="select_all">Select all</string>
-    <string name="app_details">App Details</string>
-    <string name="package_name">Package Name</string>
+    <string name="app_details">App details</string>
+    <string name="package_name">Package name</string>
     <string name="uid">UID</string>
     <string name="app_name">Name</string>
     <string name="version">Version</string>
     <string name="target_sdk">Target SDK</string>
     <string name="installed_on">Installed on</string>
-    <string name="last_update">Last Update</string>
+    <string name="last_update">Last updated</string>
     <string name="permissions">Permissions</string>
-    <string name="root_app_info">System daemons running with maximum privileges.</string>
-    <string name="netd_app_info">The system daemon which performs the DNS resolutions.</string>
+    <string name="root_app_info">Running system daemons with max privileges.</string>
+    <string name="netd_app_info">The system daemon performing the DNS resolutions.</string>
     <string name="android_app_info">The Android system.</string>
     <string name="unknown_app_info">\"Unknown\" marks connections whose app could not be determined.</string>
-    <string name="packets_dropped">Packets Dropped</string>
+    <string name="packets_dropped">Packets dropped</string>
     <string name="capture_prefs">Capture</string>
-    <string name="pcapdroid_trailer">PCAPdroid Trailer</string>
-    <string name="pcapdroid_trailer_summary">A trailer containing additional metadata, such as the app name, will be added to the dumped packets.</string>
+    <string name="pcapdroid_trailer">PCAPdroid trailer</string>
+    <string name="pcapdroid_trailer_summary">A trailer containing additional metadata (such as the app name) will be added to the dumped packets.</string>
     <string name="trailer_how_to">How to display the trailer data in Wireshark</string>
-    <string name="remove_ads">Remove Ads</string>
+    <string name="remove_ads">Remove ads</string>
     <string name="pending_transaction">Please complete your transaction to redeem this feature</string>
-    <string name="purchased_feature_ok">The purchased feature is now available!</string>
+    <string name="purchased_feature_ok">You can now use the purchased feature.</string>
     <string name="ctrl_consent">An app wants to capture your device traffic.\nDeny it if you are not sure!</string>
     <string name="deny">Deny</string>
     <string name="allow">Allow</string>
-    <string name="ctrl_consent_title">PCAPdroid Control Request</string>
+    <string name="ctrl_consent_title">PCAPdroid control request</string>
```

Comment:
```suggestion
    <string name="ctrl_consent_title">Request to control PCAPdroid </string>
```

Reply:
This is the title of a dialog, let's keep "PCAPdroid control request" for now